src/plugins/intel_cpu/src/nodes/executors/shl/shl_fullyconnected.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shl_fullyconnected.hpp"
+
+#include "csinn/csi_nn.h"
+#include "nodes/executors/executor.hpp"
+#include "nodes/executors/memory_arguments.hpp"
+#include "utils/debug_capabilities.h"
+
+namespace ov {
+namespace intel_cpu {
+
+bool ShlFCExecutor::supports(const FCConfig& config) {
+    if (config.attrs.weightsNonTransposed) {
```

Comment:
Shouldn't we disable `FuseFCAndTransposeOnWeights` pass on graph_optimizer level then to prevent fallback on ref impl?

Reply:
I thought about that and left the [comment](https://github.com/openvinotoolkit/openvino/pull/23964#issuecomment-2196284016) to hear opinion of other reviewers.
Disabled, thanks!